meinberlin/apps/polls/assets/PollManagement.jsx

```diff
@@ -173,20 +172,23 @@ export const PollManagement = (props) => {
     },
     popperMenuItems: [
       {
-        styleClass: 'btn btn--light btn--small',
+        styleClass: 'btn btn--light btn--small submenu-item__first',
         text: 'vorgegebene Antworten',
```

Comment:
better use English and we then translate by using django.gettext.. you can just use your favourite english wording as mB is only in German :)